omero/conf.py

```diff
@@ -74,6 +74,7 @@
 .. |iceversion| replace:: 3.6.5
 .. |postgresversion| replace:: 11
 .. |javaversion| replace:: 11
```

Comment:
Is this effectively `javaversion_recommended`?

Reply:
For the server installation yes it is.
The value was used for both clients and servers section leading to the confusion.
I will rename the variable so it is more explicit and to avoid misusage in the future